napari/layers/labels/labels.py

```diff
@@ -320,6 +349,7 @@ def _get_state(self):
                 'properties': self._properties,
                 'seed': self.seed,
                 'data': self.data,
+                'color_dict': self.color_dict,
```

Comment:
Is it important to save the color mode as well? A `Labels` layer from the state output here would always be in the color dict mode, even if the viewer was in the mode where the colors are automatically assigned. This would reproduce the displayed image, but the interactivity would be different (e.g., layer controls would be different on GUI).

Reply:
Yup agreed, that's a little funny. Right now we don't have color mode in the points layer serialized. One reason is we didn't add it to the main API. I'd be fine merging without for now and then revisiting.

Reply:
Tests are now failing because `color_mode` was added to serialization and not the main API. I think for now I'd keep the `color_mode` off of `_get_state` which is still private and then we can fix this later, when we fix it for points too. It's easier to add this later than to expand the API now and then have to remove it later.

Reply:
Yeah, that makes sense to me!